docs/01-start-here/06-faqs.md

```diff
@@ -0,0 +1,23 @@
+#FAQs
+
+## I can see mark-up in my response, but it's not in the application anywhere. Where is it coming from?
+
+Your markup is coming from CAPI and used as-is.
+
+For example: you're trying to style a class, but can't find it in the Frontend codebase and want to make sure it's only used in the HTML where you're looking.
+
+In this case, you can check in the `content-api` repo and your class will probably appear in some text fixtures and then you can look in the `flexible-content` repo (composer) where you'll probably see the view that adds the class you're styling.
```

Comment:
Maybe mention hitting the CAPI url for the bit of content in question so you can see the html in the response? 
Or say that you can use Teleporter to view the CAPI response for the same thing.